reo/src/load_profile.py

```diff
@@ -40,6 +40,18 @@
 from shapely import geometry as g
 from reo.exceptions import LoadProfileError
 log = logging.getLogger(__name__)
+import json
+
+library_path_base = os.path.join('input_files', 'LoadProfiles')
+load_type_file_map = {"Electric": "Load8760_norm_",
+                        "SpaceHeating": "SpaceHeating8760_norm_",
+                        "DHW": "DHW8760_norm_",
+                        "Cooling": "Cooling8760_norm_"}
+
+space_heating_annual_loads = json.load(open(os.path.join(library_path_base, "space_heating_annual_mmbtu.json"), "rb"))
+dhw_annual_loads = json.load(open(os.path.join(library_path_base, "dhw_annual_mmbtu.json"), "rb"))
+total_heating_annual_loads = {city: {building: space_heating_annual_loads[city][building] + dhw_annual_loads[city][building] for building in space_heating_annual_loads[city].keys()} for city in space_heating_annual_loads.keys()}
+space_heating_fraction_flat_load = {city: space_heating_annual_loads[city]["flatload"] / total_heating_annual_loads[city]["flatload"] for city in space_heating_annual_loads.keys()}
```

Comment:
Since all of this data on lines 51-54 are loaded in when anything from this file is imported (i.e. when the API is started) then all of this data consumes RAM for all time. It looks like at least some of the data is only used conditionally so it would make more sense to only load the data conditionally - unless it is a speed bottleneck.

Reply:
I made these conditionally loading and reduced the amount of loading and calcs by creating a couple of .json files with the data.